docs/governance/charter.md

```diff
@@ -0,0 +1,42 @@
+<!-- 
+A project charter discusses _what it is and why it exists_, a governance discusses _how it operates_.
+
+Examples
+
+https://docs.clearlydefined.io/docs/community/charter
+https://openssf.org/about/charter/
+https://github.com/cncf/foundation/blob/main/charter.md
+https://github.com/mochajs/mocha/blob/main/PROJECT_CHARTER.md
+https://github.com/nodejs/TSC/blob/main/TSC-Charter.md
+Combined charter - governance https://github.com/camaraproject/Governance/blob/main/ProjectCharter.md
+blog about charters https://opensource.org/blog/what-is-open-governance-drafting-a-charter-for-an-open-source-project
+-->
+
+# Project Charter
+
+## 1. Mission
+<!-- Describe the project's purpose and the problem it addresses. Include a short mission statement. -->
+The EESSI project aims to build a common software stack that is:
+- Cross-platform (laptop, Cloud VM, HPC Cluster)
+- Ready-to-use (served over the internet, just mount-and-go)
+- Optimized for a wide range of hardware architectures (CPU, GPU, interconnects)
+- Easily extendable
+
+## 2. Scope
+<!-- Define what is within the scope of the project and what is explicitly out of scope. -->
+EESSI will focus on creating a repository of software installations (software.eessi.io). This requires:
+- code to build and deploy new software into the repository
+- code to make EESSI work on end-user systems
+- infrastructure to build new software for software.eessi.io
+- infrastructure to host the repository for software.eessi.io
+
+All of these (both code and infrastructure itself) are considered 'in scope' for the project.
+
+There are additional repositories under the eessi.io namespace, such as dev.eessi.io and riscv.eessi.io. All code related to those repositories is also considered part of the EESSI project. However, the repositories themselves (i.e. their content) and the infrastructure hosting them is not.
```

Comment:
```suggestion
There are additional CernVM-FS repositories under the `eessi.io` namespace, such as `dev.eessi.io` and `riscv.eessi.io`.
All code related to those repositories is also considered part of the EESSI project.
However, the repositories themselves (i.e. their content) and the infrastructure hosting them is not.
```

Reply:
Agreed, but we should say this in the governance (the charter just discusses what is part of  the EESSI open source project - and we consider this part of EESSI, so I now included it).